python/modules/SoftBDeepAK8SFProducer.py

```diff
@@ -107,6 +119,24 @@
     }
 }
 
+DeepW_Fake_SF = {
+    "2016" : {
+        (200, 300) :  (1.161, 0.032),
```

Comment:
I thought we will go for single value for W mistag as well. Any reason?

Reply:
the values seem to vary more than their statistical uncertainty for W